apps/communications/contacts/js/match_service.js

```diff
@@ -9,6 +9,13 @@
   var extensionFrame = document.createElement('iframe');
   extensionFrame.id = 'match-extension';
   document.body.appendChild(extensionFrame);
+
+  if (document.querySelector('#fb-curtain') == null) {
```

Comment:
Why dont we move this logic to https://github.com/mozilla-b2g/gaia/blob/master/shared/pages/import/js/curtain.js#L9 ? Seems that there is a collision there.